app/(tabs)/auth/registerPage.tsx

```diff
@@ -0,0 +1,122 @@
+import { OrangeButton } from '@/components/OrangeButton'
+import Colors from '@/constants/Colors'
+import { useAuth } from '@/providers/AuthProvider'
+import { router } from 'expo-router'
+import React, { useState } from 'react'
+import { StyleSheet, Text, TextInput, View } from 'react-native'
+
+export default function RegisterScreen() {
+    const { onRegister } = useAuth()
+    const [username, setUsername] = useState('')
+    const [password, setPassword] = useState('')
+    const [confirmPassword, setConfirmPassword] = useState('')
+    const [error, setError] = useState('')
+
+    const handleRegister = async () => {
+        if (password !== confirmPassword) {
+            setError('Passwords do not match.')
+            return
+        }
+        try {
+            console.log('Registering...')
+            const response = await onRegister(username, password)
+            if (response) {
+                console.log('Registration successful')
+                router.push('/auth/loginPage')
+                setError('')
```

Comment:
Analogicznie do `loginPage` - czy potrzebne są te wywołania?

Reply:
absolutnie nie, już zrobione to inaczej